devon4ng-pl/Jenkinsfile

```diff
@@ -0,0 +1,182 @@
+@Library('ProductionLineTemplateLib')
+
+import com.capgemini.productionline.configuration.*
+
+Jenkins jenkinsConfiguration = new Jenkins();
+GitLab gitlabConfiguration = new GitLab(this, params.GITLAB_USER_PRIVATE_TOKEN, ProductionLineGlobals.GITLAB_BASE_URL);
+
+ 
+
+pipeline{
+
+    agent any
+
+  tools {
+        nodejs "NodeJS LTS"
```

Comment:
So this is a requirement ?
Then we need to add this to the documentation.

Reply:
sure, will be added